src/explore-education-statistics-admin/src/services/importStatusBauService.ts

```diff
@@ -0,0 +1,21 @@
+import client from '@admin/services/utils/service';
+
+export interface ImportStatusBau {
```

Comment:
Would just call this `ImportStatus` as there's no reason services need to specific to BAU responsibilities. Rename the file accordingly as well.

Reply:
This could lead to frontend/backend confusion, as there is already a ImportStatus on the backend which is different to what I'm referencing here. Can still change it if you think that isn't an issue?

Reply:
Don't think there's much of an issue here. The backend and frontends don't need to be one-to-one mappings of one another in terms of naming.

I actually think `ImportStatusBau` in the long term could probably just be consolidated into `ImportStatus` anyway.